lang/cs/Org.Apache.REEF.Client/Local/LocalClient.cs

```diff
@@ -86,7 +90,36 @@ public sealed class LocalClient : IREEFClient
             // Intentionally left blank.
         }
 
-        public void Submit(IJobSubmission jobSubmission)
+        private string CreateBootstrapAvroConfig(IJobSubmission jobSubmission, string driverFolder)
+        {
+            var paramInjector = TangFactory.GetTang().NewInjector(jobSubmission.DriverConfigurations.ToArray());
+
+            var bootstrapArgs = new AvroBootstrapArgs
+            {
+                jobSubmissionFolder = driverFolder,
+                jobId = jobSubmission.JobIdentifier,
+                tcpBeginPort = paramInjector.GetNamedInstance<TcpPortRangeStart, int>(),
+                tcpRangeCount = paramInjector.GetNamedInstance<TcpPortRangeCount, int>(),
+                tcpTryCount = paramInjector.GetNamedInstance<TcpPortRangeTryCount, int>(),
+            };
+
+            var avroLocalBootstrapArgs = new AvroLocalBootstrapArgs
+            {
+                sharedBootstrapArgs = bootstrapArgs,
+                numberOfEvaluators = _numberOfEvaluators
+            };
+
+            var submissionArgsFilePath = Path.Combine(driverFolder, "argsfile.json");
```

Comment:
Also, maybe the driver folder is not the best place for this, as the parameters are for the client, not the Driver, right? Or will the same file be used once we move the java configuration generation to the Driver?